tester/src/test/java/com/silanis/esl/sdk/examples/GroupRetrievalExampleTest.java

```diff
@@ -0,0 +1,21 @@
+package com.silanis.esl.sdk.examples;
+
+import com.silanis.esl.sdk.examples.GroupRetrievalExample;
+import org.junit.Test;
+
+import static junit.framework.TestCase.assertTrue;
+
+public class GroupRetrievalExampleTest {
+
+    @Test
+    public void verifyResult() {
+        GroupRetrievalExample example = new GroupRetrievalExample();
+        example.run();
+
+        assertTrue(example.getThirdGroup().get(0).getName().equalsIgnoreCase("third_Group"));
+        assertTrue(example.getThirdGroup().size() == 1);
```

Comment:
Do not use assertTrue in this case. Use assertThat and Matcher( hasSize in this case)